DotNet/overrides.js

```diff
@@ -63,4 +63,5 @@
 	'should send custom batch properties': {skipEmit: true},
 	'adopted styleSheets on chrome': {skipEmit: true},
 	'adopted styleSheets on firefox': {skipEmit: true},
+	'should send agentRunId with vg': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8370302)
```suggestion
	"should send agentRunId with vg": {skipEmit: true},
```